datasets/skab/README.md

```diff
@@ -0,0 +1,121 @@
+
+# About SKAB [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/waico/SKAB/graphs/commit-activity) [![DOI](https://img.shields.io/badge/DOI-10.34740/kaggle/dsv/1693952-blue.svg)](https://doi.org/10.34740/KAGGLE/DSV/1693952) [![License: GPL v3.0](https://img.shields.io/badge/License-GPL%20v3.0-green.svg)](https://www.gnu.org/licenses/gpl-3.0.html)
+We propose the Skoltech Anomaly Benchmark (SKAB) designed for evaluating the anomaly detection algorithms. SKAB allows working with two main problems (there are two markups for anomalies):
+1. Outlier detection (anomalies considered and marked up as single-point anomalies);
+2. Changepoint detection (anomalies considered and marked up as collective anomalies).
+
+SKAB consists of the following artifacts:
+1. [Datasets](#datasets);
+2. [Leaderboards](#leaderboards) for oultier detection and changepoint detection problems;
+3. Python [modules](https://github.com/waico/SKAB/blob/master/utils/evaluating.py) for algorithms’ evaluation;
+4. Python [notebooks](#notebooks) with anomaly detection algorithms.
+
+The IIot testbed system is located in the Skolkovo Institute of Science and Technology (Skoltech).
+All the details regarding the testbed and the experimenting process are presented in the following artifacts:
+- Position paper (*currently submitted for publication*);
+- [Slides](https://drive.google.com/open?id=1dHUevwPp6ftQCEKnRgB4KMp9oLBMSiDM) about the project.
+
+<a name="datasets"></a>
+# Datasets
+The SKAB v0.9 corpus contains 35 individual data files in .csv format. Each file represents a single experiment and contains a single anomaly. The dataset represents a multivariate time series collected from the sensors installed on the testbed. The [data](data/) folder contains datasets from the benchmark. The structure of the data folder is presented in the [structure](./data/README.md) file. Columns in each data file are following:
+- `datetime` - Represents dates and times of the moment when the value is written to the database (YYYY-MM-DD hh:mm:ss)
+- `Accelerometer1RMS` - Shows a vibration acceleration (Amount of g units)
+- `Accelerometer2RMS` - Shows a vibration acceleration (Amount of g units)
+- `Current` - Shows the amperage on the electric motor (Ampere)
+- `Pressure` - Represents the pressure in the loop after the water pump (Bar)
+- `Temperature` - Shows the temperature of the engine body (The degree Celsius)
+- `Thermocouple` - Represents the temperature of the fluid in the circulation loop (The degree Celsius)
+- `Voltage` - Shows the voltage on the electric motor (Volt)
+- `RateRMS` - Represents the circulation flow rate of the fluid inside the loop (Liter per minute)
+- `anomaly` - Shows if the point is anomalous (0 or 1)
```

Comment:
Why are these anomaly labels not used during experiment for the model's evaluation?